docs/modules/guides/pages/pipeline-with-proxy.adoc

```diff
@@ -0,0 +1,85 @@
+= Example of pipeline script with starting VIVIDUS proxy in DOCKER container
```

Comment:
Do you mind to put some comments about the sections of the file?

Here is an example 

https://github.com/vividus-framework/vividus/blame/03bfc123f0cd96b32dbd3b6b7f94bcaca3be283a/docs/modules/ROOT/pages/tests-configuration.adoc#L106-L132
![image](https://user-images.githubusercontent.com/17816729/130621041-310b044c-3562-473f-ae6c-77332736883f.png)